content/work/index.rst

```diff
@@ -51,26 +48,21 @@ services:
       - User experience and interface design
       - Automated, living style guides
       - Brand design and documentation
+summary: |
+  **Do you need help developing a web application?**
+  We provide extensive experience in *product development*,
+  *branding*, *user-experience design*,
```

Comment:
we say experience twice ... can we change the first one to expertise? or history? or change the second to human-centered design?